src/search_datastructures.hpp

```diff
@@ -430,6 +430,12 @@ class GlobalDataHolder
     size_t              queryTotal = 0;
     std::atomic<size_t> queryCount = 0;
     size_t              records_per_batch = 100;
+
+    GlobalDataHolder() = default;
```

Comment:
Is there a particular reason why we did not have them before?

Reply:
The compiler generates defaults. I just thought it's good to explicitly add them and delete the ones that shouldn't be used anyway.